pom.xml

```diff
@@ -354,6 +354,9 @@ THE SOFTWARE.
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-javadoc-plugin</artifactId>
           <version>2.10.3</version>
+          <configuration>
+            <quiet>true</quiet>
```

Comment:
Why does it need to be kept verbose exactly? Did you read my commit comment for 31cd68c?